Bottom: 5bac3d07b97e5f74593b3dc65ac3617850cebce5
Top:    5ca230af279b112e988f817c05b3591a0a0e8c0a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 01:28:23 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 92c8b35..765aeec 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2747,7 +2747,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
